test/e2e/azure_net_pol.go

```diff
@@ -177,27 +183,32 @@ func AzureNetPolSpec(ctx context.Context, inputGetter func() AzureNetPolSpecInpu
 
 	By("Applying a network policy to deny ingress access to app: webapp, role: backend pods in development namespace")
 	nwpolicyName, namespaceE2E, nwpolicyFileName := "backend-deny-ingress", nsDev, "backend-policy-deny-ingress.yaml"
+	Log("starting to applying a network policy to deny access to app: webapp, role: backend pods in development namespace")
```

Comment:
I see that you're trying to use the test description for log statements. Would it be more useful if we can add details about the resource being applied. Eg:
`Logf("Applying network policy %s/%s", namespaceE2E, nwpolicyName)`

Reply:
Same for other places as well.